### PR TITLE
add newrelic in Procfile

### PR DESCRIPTION
**Pull request by Moomoo-Seukla**
Warning :  the MonkeyPatchWarning  be back  (see https://github.com/CanalTP/kirin/pull/425) 

ticket [ND-1759](https://jira.kisio.org/browse/ND-1759)

### CODE REVIEW

**Review by woshilapin — APPROVED** [review]
I never set up NewRelic on my machine and I'm still able to run `honcho start` without any problem with this branch and query the web API.

File: readme.md [hunk 1]
```diff
@@ -121,6 +121,11 @@ Kirin supports Python 2.7 (virtualenv is strongly advised).
     KIRIN_LOG_FORMATTER='json'  # If you wish to have logs formated as json (more details)
     PYTHONUNBUFFERED = 1
     ```
+  if using NewRelic, this line should be added in `.env`
```
Comment (woshilapin) [inline readme.md:124]: I've tested to use Newrelic with `honcho start` and this new setup and it works great. I would add a few things into the documentation to help:
-  a link to the template file [`newrelic.ini`](https://github.com/CanalTP/fab_kirin/blob/master/fabfile/templates/newrelic.ini)
- and maybe a procedure of how to replace the 2 templated variables in it
  - how to get the API key from the Newrelic interface (and which key to get)
  - some suggestion on how to configure `app_name` when testing on local (`kirin <+username+>` for example)

Comment (pbougue) [reply readme.md:124]: There's already a NewRelic section in CONTRIBUTING.md: I will move this there and add your suggestions.

**Review by pbougue — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (woshilapin) [reply readme.md]: Perfect, thanks!

**Review by woshilapin — APPROVED** [review]

**Pull request merged**